app/frontend/src/lib/__mocks__/autocomplete.js

```diff
@@ -0,0 +1,11 @@
+const view = {
+    show: jest.fn(),
+    hide: jest.fn(),
+    focus: jest.fn()
+}
+
+const autocomplete = {
+    view
+}
+
+export default autocomplete
```

Comment:
Newline. And this file looks 4 space indented

Reply:
ah this file isnt actually needed